src/GovUk.Education.ExploreEducationStatistics.Content.Services/DataSetService.cs

```diff
@@ -176,28 +176,28 @@ internal static class FreeTextReleaseFileValueResultQueryableExtensions
 {
     internal static IOrderedQueryable<FreeTextValueResult<ReleaseFile>> OrderBy(
         this IQueryable<FreeTextValueResult<ReleaseFile>> query,
-        DataSetsListRequestOrderBy orderBy,
-        SortOrder sort)
+        DataSetsListRequestSortBy sortBy,
```

Comment:
Shouldn't the name here be `sort` rather than `sortBy`?

It doesn't really matter because it's the internal implementation of the service, but this method could also be renamed from `OrderBy` to `Sort`.